pkg/metrics/features/metrics.go

```diff
@@ -194,6 +208,18 @@ var (
 		actionAdd,
 		actionDel,
 	}
+
+	defaultClusterMeshMode = []string{
+		advConnClusterMeshModeAPIServer,
+		advConnClusterMeshModeETCD,
+		advConnClusterMeshModeKVStoreMesh,
+		advConnClusterMeshModeAPIServerOrETCD,
```

Comment:
How about directly referencing the types defined in the clustermesh package, to avoid duplications that could eventually get out of sync (or vice versa, defining the interface here rather than in the clustermesh package to prevent the circular import)?